components/infobox/commons/infobox_league.lua

```diff
@@ -179,6 +191,63 @@ function League:_definePageVariables(args)
     self:defineCustomPageVariables(args)
 end
 
+function League:_setLpdbData(args)
+    local lpdbData = {
+        name = self.name,
+        tickername = args.tickername,
+        shortname = args.shortname,
+        banner = args.banner,
+        icon = args.icon,
+        series = args.series,
+        previous = args.previous,
+        previous2 = args.previous2,
+        next = args.next,
+        next2 = args.next2,
+        patch = args.patch,
+        endpatch = args.endpatch or args.epatch,
+        type = args.type,
+        organizers = mw.ext.LiquipediaDB.lpdb_create_json({
+            organizer1 = args.organizer or args.organizer1,
+            organizer2 = args.organizer2,
+            organizer3 = args.organizer3,
+            organizer4 = args.organizer4,
+            organizer5 = args.organizer5,
+        }),
+        startdate = Variables.varDefault('tournament_startdate', '1970-01-01'),
+        enddate = Variables.varDefault('tournament_enddate', '1970-01-01'),
+        sortdate = Variables.varDefault('tournament_enddate', '1970-01-01'),
+        location = Locale.formatLocation({city = args.city or args.location, country = args.country}),
+        location2 = Locale.formatLocation({city = args.city2 or args.location2, country = args.country2}),
+        venue = args.venue,
+        prizepool = Variables.varDefault('tournament_prizepoolusd', 0),
+        liquipediatier = Variables.varDefault('tournament_liquipediatier'),
+        liquipediatiertype = Variables.varDefault('tournament_liquipediatiertype'),
+        status = args.status,
+        format = args.format,
+        sponsors = mw.ext.LiquipediaDB.lpdb_create_json({
+            sponsor1 = args.sponsor or args.sponsor1,
+            sponsor2 = args.sponsor2,
+            sponsor3 = args.sponsor3,
+            sponsor4 = args.sponsor4,
+            sponsor5 = args.sponsor5,
+        }),
+        links = mw.ext.LiquipediaDB.lpdb_create_json({
+            discord = Links.makeFullLink('discord', args.discord),
+            facebook = Links.makeFullLink('facebook', args.facebook),
+            instagram = Links.makeFullLink('instagram', args.instagram),
+            twitch = Links.makeFullLink('twitch', args.twitch),
+            twitter = Links.makeFullLink('twitter', args.twitter),
+            website = Links.makeFullLink('website', args.website),
+            weibo = Links.makeFullLink('weibo', args.weibo),
+            vk = Links.makeFullLink('vk', args.vk),
+            youtube = Links.makeFullLink('youtube', args.youtube),
+        }),
+    }
+
+    lpdbData = self:addToLpdb(lpdbData)
```

Comment:
```suggestion
    lpdbData = self:addToLpdb(lpdbData, args)
```
we probably should add args here too

and you should define the default function, else it will error with `attempt to call method 'addToLpdb' (a nil value)`

Reply:
Ah lost in rebase